static/js/lib/courses.js

```diff
@@ -1,23 +1,23 @@
 //@flow
-import moment from "moment"
-import _ from "lodash"
-
-import type { Course } from "../flow/discussionTypes"
 import {
-  platforms,
+  COURSE_ARCHIVED,
   COURSE_AVAILABLE_NOW,
-  COURSE_PRIOR,
-  COURSE_UPCOMING
+  COURSE_CURRENT,
+  COURSE_PRIOR
 } from "./constants"
 
-export const courseAvailability = (course: Course) =>
-  course.platform === platforms.OCW
-    ? COURSE_AVAILABLE_NOW
-    : moment(course.start_date).isAfter(moment())
-      ? _.capitalize(COURSE_UPCOMING)
-      : moment(course.end_date).isBefore(moment())
-        ? _.capitalize(COURSE_PRIOR)
-        : COURSE_AVAILABLE_NOW
+import type { Course } from "../flow/discussionTypes"
+
+export const availabilityLabel = (availability: ?string) => {
```

Comment:
just a question, for similar little helper functions for the course UI we generally have the signature `Course => string`, do you think it would make sense to keep the same API here for consistency? You could add a line right below this one like

```js
const { availability } = course
```

to grab the availability value out of the course object

Reply:
This was the original signature, but I changed it because it's now being used by `SearchFacet` as the `labelFunction`, which does not get passed the full `Course` object.

Reply:
https://github.com/mitodl/open-discussions/pull/1922/files#diff-f3d2dc8b6c7c97d4acc6f60b8e663a61R77

Reply:
ah ok, that makes sense